konfera/models/ticket.py

```diff
@@ -29,6 +29,7 @@ class Ticket(KonferaModel):
     last_name = models.CharField(max_length=128)
     email = models.EmailField()
     phone = models.CharField(max_length=64, blank=True)
+    # TODO isn't the description useless? The general description of the ticket provides type.description...
```

Comment:
Please remove the TODO. TODO should mark some future functionality, or change that shall be done due to some changes, you are asking question, that shall be either discussed in issue comments, or at mailinglinst, or chat.

To answer your question:
`ticket_type` has description that is displayed when user is registering the ticket.
`ticket` has description that is filled by user. We can call `register.form.RegistrationForm` without description or with it, as for tickets, such as press, or aid we would like to get some input from attendee before confirming the ticket. Seems description isnt best name :) but attendee describe something about himself so we can re-name to something more obvious... What about about_me?